scripts/cob-shutdown

```diff
@@ -17,11 +17,11 @@ $robot_name-s1
 $robot_name-h1
 $robot_name-b1"
 
-cob-stop-core
+#cob-stop-core
 
-while /opt/ros/indigo/env.sh rostopic list > /dev/null; do
-	sleep 1
-done
+#while /opt/ros/indigo/env.sh rostopic list > /dev/null; do
+#	sleep 1
+#done
 
 for client in $client_list; do
```

Comment:
does this really wait for all pc to shutdown __before__ the robot is turned off?

Please make sure that if the base pc is the last one which shuts down.